rx-library/build.gradle

```diff
@@ -1,51 +1,15 @@
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
-apply plugin: 'org.jetbrains.dokka-android'
-
-android {
-  compileSdkVersion 26
-
-  defaultConfig {
-    minSdkVersion 15
-    targetSdkVersion 26
-    versionCode 1
-    versionName "1.0"
-
-    testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-  }
-
-  buildTypes {
-    release {
-      minifyEnabled false
-      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-    }
-  }
-
-  sourceSets {
-    main.java.srcDirs += 'src/main/kotlin'
-    test.java.srcDirs += 'src/test/kotlin'
-  }
-}
-
-dokka {
-  outputFormat = 'html'
-  outputDirectory = "$buildDir/javadoc"
-}
+apply plugin: 'kotlin'
 
 dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Do you need that line? I think that you can remove it

Reply:
Done!